test/main.js

```diff
@@ -31,6 +32,48 @@ var options = {
   templateDonut: {
     'twig': './templates/donut.twig'
   },
+  convertCategoryTitles: true,
+  convertCategoryTitlesData: {
+    "categories": {
+      "base": "00-atoms",
```

Comment:
This file is the unit test. The numbers are not for real-world use, only for tests.

The numbers were added here because as our main current use was injecting into pattern lab, I made the `convertCategoryTitles` `true` by default in the main app.